app/queries/servers/post.ts

```diff
@@ -234,9 +234,8 @@ export const observeSavedPostsByIds = (database: Database, postIds: string[]) =>
 };
 
 export const getIsPostPriorityEnabled = async (database: Database) => {
-    const featureFlag = await getConfigValue(database, 'FeatureFlagPostPriority');
```

Comment:
Unless the server somehow sends the config based on the feature flag, if I turn on the feature flag, turn on the config, and then turn off the feature flag, I would have the following scenario:
```
config: {
    ...
    PostPriority: 'true',
    FeatureFlagPostPriority: 'false',
}
```
So... we will be showing the post priority even in the case the feature flag is set to false. Is this scenario possible?

Reply:
So in this case, given that config is `true` the feature will be shown which seems good to me. This config setting is available to be modified in the System Console so it will probably be the first thing they use to disable the feature. The feature flag is not available in the System Console and it is only overriden by a env var which is more unlikely they use. 

Features that start out with feature flags are mostly to guard newly released features in case they have issues and quickly turn off in Cloud through Split.io. I wouldn't expect anyone turning it off in the Feature Flag but not the config.